exporters/committime/collector_base.py

```diff
@@ -251,6 +306,78 @@ def get_repo_from_jenkins(self, jenkins_builds):
                 )
         # If no repo is found, we will return None, which will be handled later on
 
+    def get_metrics_from_pipelineruns(
```

Comment:
Any description of what the method does is welcome :)

Reply:
I guess I got too excited about changing from WIP and forgot to comment 😆  I'll document all the methods I added.